omod/src/test/java/org/openmrs/module/fhir2/providers/r4/ConditionFhirResourceProviderWebTest.java

```diff
@@ -226,6 +234,56 @@ public void shouldCreateNewConditionGivenValidConditionResource() throws Excepti
 		assertThat(response.getStatus(), is(201));
 	}
 	
+	@Test
+	public void updateCondition_shouldUpdateConditionWithPath() throws Exception {
+		String conditionJson;
+		try (InputStream is = this.getClass().getClassLoader().getResourceAsStream(JSON_UPDATE_CONDITION_PATH)) {
+			Objects.requireNonNull(is);
+			conditionJson = IOUtils.toString(is, StandardCharsets.UTF_8);
+		}
+		Condition condition = new Condition();
+		condition.setIdBase(CONDITION_UUID);
+		when(conditionService.update(conditionJson, any(Condition.class))).thenReturn(condition);
+		
+		MockHttpServletResponse response = put("/Condition/" + CONDITION_UUID).jsonContent(conditionJson)
+		        .accept(FhirMediaTypes.JSON).go();
+		
+		assertThat(response, isOk());
+	}
+	
```

Comment:
Please add a unit test for nonExistentCondition-
`updateCondition_shouldThrowErrorForNonExistentCondition`